simulation/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(SUBSYS_DEPS common io surface kdtree features search octree visualization fi
 
 set(build FALSE)
 find_package(OpenGL)
-find_package(GLEW)
+find_package(GLEW CONFIG)
```

Comment:
Can't GLEW be found, if you just use the original `find_pacakge(GLEW)` ?

Else you need to keep the:
```
if(APPLE)
  find_pacakge(glew CONFIG)
ELSE()
  find_pacakge(GLEW)
ENDIF()
```

Reply:
It can, but it's incorrect. The package comes with links to non-existing locations in my setup.